vscode/src/non-stop/FixupController.ts

```diff
@@ -59,31 +59,49 @@ export class FixupController
             vscode.commands.registerCommand('cody.fixup.accept-all', () => this.acceptFixups()),
             vscode.commands.registerCommand('cody.fixup.diff', treeItem => this.showDiff(treeItem)),
             vscode.commands.registerCommand('cody.fixup.codelens.cancel', id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'cancel' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'cancel', hasV2Event: true })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.cancel', 'clicked', {})
                 return this.cancel(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.diff', id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'diff' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'diff', hasV2Event: true })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.diff', 'clicked', {})
                 return this.diff(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.retry', async id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'regenerate' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', {
+                    op: 'regenerate',
+                    hasV2Event: true,
+                })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.retry', 'clicked', {
+                    privateMetadata: { op: 'regenerate' },
+                })
                 return this.retry(id)
             }),
             vscode.commands.registerCommand('cody.fixup.codelens.undo', id => {
-                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'undo' })
+                telemetryService.log('CodyVSCodeExtension:fixup:codeLens:clicked', { op: 'undo', hasV2Event: true })
+                telemetryRecorder.recordEvent('cody.fixup.codeLens.undo', 'clicked', {})
```

Comment:
If there is nothing in parameters you can omit it entirely:

```suggestion
                telemetryRecorder.recordEvent('cody.fixup.codeLens.undo', 'clicked')
```